Source/JavaScriptCore/runtime/JSGlobalObject.h

```diff
@@ -604,6 +610,8 @@ class JSGlobalObject : public JSSegmentedVariableObject {
     std::optional<unsigned> m_stackTraceLimit;
     Weak<FunctionExecutable> m_executableForCachedFunctionExecutableForFunctionConstructor;
 
+    RequiresTrustedTypes m_requiresTrustedTypes { RequiresTrustedTypes::Enforced };
```

Comment:
Why do we default to enforcing when the feature is off by default in UnifiedWebPreferences?

Reply:
The feature is on by default fwiw. I don't think this default makes a difference either way, but just to be safe I'll set it to NotRequired